problems/arraysWithLoops.js

```diff
@@ -66,15 +103,25 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+  let secondSmallest = nums[0]
+  for(let i = 0; i <= nums.length -1; i++)
+  if (nums[i] <= secondSmallest) secondSmallest = nums [i]
+  return secondSmallest +1
+}
 
 /**
  * Takes in array and returns the second largest number in the array
  * @param {number[]} nums
  * @returns {number} second largest value
  */
 
-function findSecondLargest() {}
+function findSecondLargest(nums) {
+  let secondLargest = nums [0]
+  for (let i = 0; i <= nums.length - 1; i++)
+  if (nums[i] >= secondLargest) secondLargest = nums [i]
```

Comment:
Same goes here. Make sure you are keep track of the first largest AND the second largest.